.github/workflows/source-build.yaml

```diff
@@ -0,0 +1,62 @@
+name: Source container build
+
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - source-container-build/**
+
+  pull_request:
+    branches:
+    - main
+    paths:
+    - source-container-build/**
+
+env:
+  REGISTRY: quay.io/redhat-appstudio
+  IMAGE_NAME: build-definitions-source-image-build-utils
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+    - name: Run checks
+      run: |
+        sudo apt-get install -y python3-pip
```

Comment:
Will it be good to use `[setup-python]( https://github.com/actions/setup-python?tab=readme-ov-file#basic-usage)` github action here and then do the `pip install`, so that we can upgrade python version in this CI in future if needed?

Reply:
Probably. The Python version is not a big deal for this step. When we need that, we can change it then.